packages/react-composites/src/composites/common/icons.tsx

```diff
@@ -18,6 +18,8 @@ import React from 'react';
  * @public
  */
 export const COMPOSITE_ONLY_ICONS = {
+  lobbyScreenConnectingToCall: <>☕</>,
```

Comment:
what is the emoji support like on all platforms? should we update this to a fluent icon? e.g. https://iconcloud.design/search/filter/Fluent%20System%20Library/Fluent%20Regular/coffee/b1c9959dd-3821b375b

Reply:
@pereiralex